README.md

```diff
@@ -1,2 +1,22 @@
-# rebar3-nix-bootstrap
-Shim command to help bootstrap a rebar3 project on Nix
+Rebar3 Nix Bootstrap
+====================
+
+This is a very simply 'shim' project that exists to help rebar3 work
+in the context of Nix. Its purpose is to bootstrap a project so that
+rebar3 doesn't try to manage dependencies (that should be left to
+Nix).
+
+This does the following work.
+
+# It copies the hex registry to a place where rebar3 can find it
+# It copies any dependencies into the `_build` dir to a place where
```

Comment:
Is it possible to make it link things, not copy? Ideally there is no need to create copies and reuse files from /nix/store/